src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid $card-border-color;
+  border-radius: $card-border-radius;
+  @include box-sizing-border-box;
+
+  width: $card-width;
+  padding: $card-padding;
+
+  line-height: 18px;
+  color: $main-text-color;
+
+  @include transition(transform);
+
+  &__title {
+    @include resetMargin(margin, 0 0 4px);
+    font-size: $card-font-size-title;
+    font-weight: 500;
+    @include transition(color);
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    .card__title {
+      color: $card-titlte-hover-color;
+    }
+  }
+
+  &__product-img {
+    align-self: center;
+    display: block;
+
+    @include resetMargin(margin-bottom, 40px);
+
+    height: $card-img-height;
+    width: $card-img-width;
+  }
+
+  &__product-code {
+    @include resetMargin(margin, 0 0 16px);
+
+    font-size: $card-font-size-product-description;
+    line-height: $card-line-height;
+    color: $card-sub-text-color;
+  }
+
+  &__stars {
+    @include flex-space-between;
+    @include resetMargin(margin-bottom, 24px);
+  }
+
+  &__price {
+    @include flex-space-between;
+    @include resetMargin(margin-bottom, 16px);
+
+    &-text {
+      @include resetMargin(margin, 0);
+
+      font-size: $card-font-size-title;
+      color: $card-sub-text-color;
+    }
+
+    &-amount {
+      @include resetMargin(margin, 0);
+
+      font-size: $card-font-size-amount;
+      font-weight: $font-weight-bold;
+    }
+  }
```

Comment:
Not sure about that. Usually we use Sass nesting for elements or modificators. `&-text` is not an element, but `&__price-text` is, the same for the `&-amount`.
```
&__price {

}

&__price-text {

}

&__price-amount {

}
```